test/e2e/volumes.go

```diff
@@ -76,7 +75,6 @@ func DeleteCinderVolume(name string) error {
 	return err
 }
 
-
 // These tests need privileged containers, which are disabled by default.  Run
 // the test with "go run hack/e2e.go ... --ginkgo.focus=[Feature:Volumes]"
 var _ = framework.KubeDescribe("Volumes [Feature:Volumes]", func() {
```

Comment:
I can move the `[Volume]` label up from the tests within the suite to the describe line.  `[Feature: ... ]` is used by the CI to identify tests with non-default requirements and could cause side effects if removed:

>  // These tests need privileged containers, which are disabled by default.  Run
 // the test with "go run hack/e2e.go ... --ginkgo.focus=[Feature:Volumes]"